### PR TITLE
Prepare the 2.1.105 release

### PR DESCRIPTION
**Pull request by hrfuller**

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1,6 +1,47 @@
...
+was not always respected based on the condition of the build environment,
+and the value of the PEX_ROOT.
+
+* Fail on atomic_directory work_dir collision. (#1905) 
+  `PR #1905 <https://github.com/pantsbuild/pex/pull/1905>`_
+
+* Use raw_pex_root when constructing sh_boot pexes. (#1906)
+  `PR #1906 <https://github.com/pantsbuild/pex/pull/1906>`_
+
+* Make Virtualenv more robust. (#1911)
```
Comment (jsirois) [inline CHANGES.rst:22]: I'd nix this.

File: CHANGES.rst [hunk 2]
```diff
@@ -1,6 +1,47 @@
...
+* Make Virtualenv more robust. (#1911) 
+  `PR #1911 <https://github.com/pantsbuild/pex/pull/1911>`_
+
+
+Docs.
+
+* Add support for offline downloads (#1898)
+  `PR #1898 <https://github.com/pantsbuild/pex/pull/1898>`_
+
+Various improvements to Windows support.
```
Comment (jsirois) [inline CHANGES.rst:31]: And I'd nix from here down.

**Review by jsirois — APPROVED** [review]
Some entries I'd kill. I tend to only put user-visible changes in the changelog and those are only bug-fixes, dcumentation changes / fixes and CLI changes (additions).

File: CHANGES.rst [hunk 3]
```diff
@@ -1,6 +1,47 @@
...
+-------
+
+This is a fix release which addresses issues related to build time work_dir creation,
+virtualenv and sh_boot support, as well as Windows compatibility. Also includes various CI fixes and improvements.
+
+In the unlikely event of a UUID collision in atomic workdir creation,
+pex could overwrite an existing directory and cause a corrupt state.
+When building a shell bootable ``--sh-boot`` pex the ``--runtime-pex-root``
+was not always respected based on the condition of the build environment,
+and the value of the PEX_ROOT.
```
Comment (jsirois) [inline CHANGES.rst:14]: Per the nixes below I'd also adjust this to only refer to publicly visible things. I consider Windows not real until its real. We still definitely do not support Windows at all. Itall or nothing.

File: CHANGES.rst [hunk 4]
```diff
@@ -1,6 +1,47 @@
 Release Notes
 =============
 
+2.1.105
+-------
+
+This is a fix release which addresses issues related to build time work_dir creation,
+virtualenv and sh_boot support, as well as Windows compatibility. Also includes various CI fixes and improvements.
```
Comment (Eric-Arellano) [inline CHANGES.rst:8]: @jsirois maybe nix Windows as well?

Comment (jsirois) [reply CHANGES.rst:8]: Yeah - cokes are owed.

**Review by jsirois — APPROVED** [review]

**Pull request merged**